gui/public/i18n/en-x-owo/translation.ftl

```diff
@@ -280,9 +280,9 @@ settings-general-fk_settings-skeleton_settings-description = toggwe skeweton sew
 settings-general-fk_settings-skeleton_settings-extended_spine = ewtended spine
 settings-general-fk_settings-skeleton_settings-extended_pelvis = ewtended pelvis
 settings-general-fk_settings-skeleton_settings-extended_knees = ewtended neey
-settings-general-fk_settings-vive_emulation-title = Vive emulation
-settings-general-fk_settings-vive_emulation-description = Emulate the waist tracker problems that Vive trackers have. This is a joke and makes tracking worse.
-settings-general-fk_settings-vive_emulation-label = Enable Vive emulation
+settings-general-fk_settings-vive_emulation-title = vive emuwation
+settings-general-fk_settings-vive_emulation-description = emuwate the waist twackew pwobwems that vive twackews have. This is a joke and makes twacking wowse eheh ^^.
```

Comment:
doesnt have enough owo (maintains caps on This also)

Reply:
OwO